packages/components/psammead-useful-links/src/index.jsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { grid } from '@bbc/psammead-styles/detection';
+import { getPica } from '@bbc/gel-foundations/typography';
+import {
+  GEL_SPACING, // 8 px
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+export const UsefulLink = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSerifMedium(service)};
+  color: ${C_EBON};
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+`;
+
+export const UsefulLinksUl = styled.ul.attrs({ role: 'list' })`
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    @supports (${grid}) {
+      display: grid;
+      grid-template-columns: auto auto;
+    }
+  }
+`;
+
+export const UsefulLinksLi = styled.li.attrs({ role: 'listitem' })`
+  padding-top: ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    @supports not (${grid}) {
```

Comment:
Is there another style that needs to be applied if `grid` is not supported? If not then i think you might not need this check.

Reply:
`display` and `min-width` are the only styles to apply if grid is not supported. However, I don't want them to be applied in case grid is supported, so that is why I added a condition.